src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` if the operation's success can be determined by exceptions or other means. Returning a `Boolean` might not be necessary.